packages/server/src/sdk/app/tables/duplicate.ts

```diff
@@ -0,0 +1,64 @@
+import { Table, WithoutDocMetadata } from "@budibase/types"
+import { create } from "./create"
+import { getAllInternalTables } from "./getters"
+
+/**
+ * Generates a unique name for a duplicated table by checking existing table names
+ * and adding an incrementing number if needed
+ * @param originalName - The name of the original table
+ * @param existingTables - Array of existing tables to check against
+ * @returns A unique table name
+ */
+function generateUniqueName(
```

Comment:
We have this util already (or similar enough) in `packages/shared-core/src/helpers/duplicate.ts`. Can it be reused?